tsconfig.json

```diff
@@ -0,0 +1,70 @@
+{
+  "compilerOptions": {
+    /* Visit https://aka.ms/tsconfig.json to read more about this file */
+
```

Comment:
I don't know much about those settings, maybe someone from team can check if it's ok?

Reply:
Looks good for me, I like the strict rules